entity/pokemon.go

```diff
@@ -0,0 +1,17 @@
+package entity
+
+type Pokemon struct {
```

Comment:
Same as another comment; add a comment above the definition in order to make it easier for other devs (and yourself) to know at a glance what is this about.

```suggestion
// Pokemon struct defines the fields of a pokemon ...
type Pokemon struct {
```